core/src/main/scala/ai/lum/odinson/ExtractorEngine.scala

```diff
@@ -142,10 +142,20 @@ class ExtractorEngine(
     val filter = filters(allowTriggerOverlaps)
     val mruIdGetter = MostRecentlyUsed[Int, LazyIdGetter](LazyIdGetter(indexSearcher, _))
 
+    def localQuery(extractor: Extractor, state: State): OdinResults = {
```

Comment:
if this is where results are added, can we rename to something like `queryAndAddToState` (or something less verbose but still indicating that this is the place where we add things to the state)?
I sometimes have a hard time finding the state-adding, I forget where it is and look in the `extract` method, but it's not there (and that may be because it's a `for... yield` loop), else we could maybe add a comment in the extract loop...?
(Feel **FREE** to ignore if I'm being dumb)

Reply:
Of course.  Local was a hint to look nearby but it should say what as well as where.